cumulusci/tasks/push/tasks.py

```diff
@@ -194,9 +195,14 @@ def _report_push_status(self, request_id):
 class SchedulePushOrgList(BaseSalesforcePushTask):
 
     task_options = {
+        "csv": {"description": "The CSV file to read.", "required": False},
```

Comment:
Let's match the `orgs` description:
```suggestion
        "csv": {"description": "The path to a CSV file to read.", "required": False},
```